ci/requirements/docs.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
     # Optional dependencies
     - contextily
     - geopandas
+    - panel
```

Comment:
Please move `panel` to the "Dev dependencies (building documentation)" category.

Reply:
This is updated in commit https://github.com/GenericMappingTools/pygmt/pull/2498/commits/cea4021e071f86f28b37e9294f47e0da06852137.